<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportVaccineJournalSetup</class>
 <widget class="QDialog" name="ReportVaccineJournalSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры отчета</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>&amp;Подразделение</string>
     </property>
     <property name="buddy">
      <cstring>cmbOrgStructure</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата &amp;окончания периода</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblSex">
     <property name="text">
      <string>Пол</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QSpinBox" name="edtAgeTo">
     <property name="maximum">
      <number>150</number>
     </property>
     <property name="value">
      <number>150</number>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Дата &amp;начала периода</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="edtAgeFrom">
     <property name="maximum">
      <number>150</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblAgeFrom">
     <property name="text">
      <string>Возраст с</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="lblAgeTo">
     <property name="text">
      <string>по</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QComboBox" name="cmbVaccineType"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lblInfection">
     <property name="text">
      <string>Инфекция</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="3">
    <widget class="CRBComboBox" name="cmbInfection"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblVaccineType">
     <property name="text">
      <string>Тип прививки</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cmbSex">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>М</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ж</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblVaccine">
     <property name="text">
      <string>Вакцина</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="CRBComboBox" name="cmbVaccine"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportVaccineJournalSetup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportVaccineJournalSetup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
